Google.PowerShell/Provider/GcsContentWriter.cs

```diff
@@ -40,7 +41,13 @@ public void Seek(long offset, SeekOrigin origin)
 
         public void Close()
         {
+#if !CORECLR
```

Comment:
Would it make sense to just always call `Dispose`? I assume that `Dispose` calls `Close` under the hood?